packages/backend/src/services/authentication/authentication.hooks.ts

```diff
@@ -32,10 +30,8 @@ export default {
           const spotifyAccessToken = data.access_token;
           const spotifyRefreshToken = data.refresh_token;
 
-          await app.service('users').patch(userId, { spotifyAccessToken, spotifyRefreshToken });
+          await ctx.app.service('users').patch(userId, { spotifyAccessToken, spotifyRefreshToken });
```

Comment:
Is the parameter 'ctx' named by us? If yes, we should it rename for readability

Reply:
It stands for 'context'. It's a pretty common syntax.

But we could rename it.

Reply:
Good point. I will rename it to `context` as the official feathers [documentation](https://docs.feathersjs.com/guides/basics/hooks.html#quick-example) uses that as well.

Reply:
Thank you very much 😄